python_modules/dagster/dagster/tutorials/intro_tutorial/test/test_config.py

```diff
@@ -0,0 +1,38 @@
+# encoding: utf-8
```

Comment:
what's the encoding: thing about?

Reply:
py 2.7 compatibility, we could go as deep as we want on this particular horror

Reply:
rgr. comment might be nice